packages/sdk/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## [1.0.0-rc.1`] - 2024-11-01
```

Comment:
Did you mean to add a backtick before the 1.0.0? Currently just has the trailing tick

Reply:
Just a typo 😂 
Fixed now